.github/workflows/ci-join.yaml

```diff
@@ -45,6 +46,6 @@ jobs:
         uses: actions/upload-artifact@v4
         if: always()
         with:
-          name: failed-logs
+          name: docker-logs
```

Comment:
These logs are uploaded on successful runs as well, so the label might be misleading for successful jobs.